view.php

```diff
@@ -26,9 +31,83 @@
 	<script src="libs/jquery.slim.min.js"></script>
 	<script src="libs/popper.min.js"></script>
 	<script src="libs/bootstrap.min.js"></script>
+	<script src="libs/autoComplete.min.js"></script>
 	<script src="libs/leaflet.js"></script>
 </head>
 <body class="pb-4">
+	<script>
+		window.onload = function () {
+			let categories = {};
+			fetch("get-categories.php").then((source) => source.json()).then(function(data) {
+				categories = data;
```

Comment:
Could just do `categories = <?=json_encode($db->query("...", [...]))?>` here instead of fetch